docs/sources/panels/add-series-overrides.md

```diff
@@ -0,0 +1,43 @@
++++
+title = "Add series overrides"
+type = "docs"
+[menu.docs]
+parent = "panels"
+weight = 500
++++
+
+# Add series overrides
+
+> **Note:** This page refers to the Grafana 7.0 beta UI.
+
+The feature allows a series in a graph panel to be rendered differently from the others. You can customize display options on a per-series bases or by using regex rules. For example, one series can be given a thicker line width to make it stand out or be moved to the right Y-axis.
+
+You can add multiple series overrides.
+
+1. Navigate to the graph panel you want to add a series override to.
+2. Click the panel title and then click **Edit** or press **e**.
+3. On the Panel tab, scroll down to click **Series overrides** to expand the section.
+4. Click **Add series override**.
+5. In **Alias or regex** Type or select a series. Click in the field to see available series.
+6. Click **+** and then select a style to apply to the series. You can add multiple styles to each entry.
+   - **Bars -** 
+   - **Lines**
+   - **Line fill**
+   - **Fill gradient**
+   - **Line width**
+   - **Null point mode**
+   - **Fill below to**
+   - **Staircase line**
+   - **Dashes**
+   - **Hidden Series**
+   - **Dash Length**
+   - **Dash Space**
+   - **Points**
+   - **Point Radius**
+   - **Stack**
+   - **Color**
+   - **Y-axis**
+   - **Z-index**
+   - **Transform**
+   - **Legend**
+   - **Hide in tooltip**
```

Comment:
These are all graph options, and specific to the graph and the name "series overrides" is also specific to the graph.

Reply:
Makes sense. Could you give me the definitions of these fields either here or there, then I will merge them into the graph-panel.md PR?

Reply:
- **Bars -** show series as bar graph
   - **Lines** show series as line graph
   - **Line fill** show line graph with area fill
   - **Fill gradient** Area fill gradient amount
   - **Line width** line width
   - **Null point mode** option to ignore null values or replace with zero. Important if you want to ignore gaps in your data
   - **Fill below to** fill area between two series
   - **Staircase line** show series as a starcase line 
   - **Dashes** show line with dashes
   - **Hidden Series** hide series
   - **Dash Length** Dashed line lenth
   - **Dash Space** Dashed line spacing
   - **Points** Show series as seperate points
   - **Point Radius**  Radius for point rendering
   - **Stack** Set stack group for series
   - **Color** Set series color
   - **Y-axis** Set series y-axis
   - **Z-index**  Set series z-index (rendering order). Important when overlaying different styles (bar charts, area charts)
   - **Transform**  Transform value to negative to render below y-axis
   - **Legend**  Control if a series is shown in legend 
   - **Hide in tooltip** Control if a series is shown in graph tooltip

Reply:
Great stuff. Added to graph-panel.md.